openedx/adg/common/lib/zoom_client/client.py

```diff
@@ -0,0 +1,51 @@
+"""
+Zoom Client to manage webinars
+"""
+import json
+
+from django.conf import settings
+from rest_framework.status import HTTP_201_CREATED, HTTP_204_NO_CONTENT
+from zoomus import ZoomClient as _ZoomClient
+
+
+class ZoomClient(object):
+    """
+    Zoom class to manage webinars
+    """
+
+    def __init__(self):
+        self.zoom_client = _ZoomClient(settings.ZOOM_API_KEY, settings.ZOOM_SECRET_KEY)
+
+    def create_webinar(self, webinar):
+        """
+        Create a new webinar on zoom
+        """
+        response = self.zoom_client.webinar.create(**webinar.get_formatted_data())
+        return json.loads(response.content) if response.status_code == HTTP_201_CREATED else {}
```

Comment:
Can we pls log results of all the calls to zoom api? Pls look into mandrill client for example.

Reply:
This applies to all methods.